src/commands/support/manual.ts

```diff
@@ -0,0 +1,14 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const MANUAL_URL = 'https://cdn.discordapp.com/attachments/785976111875751956/927657411764969612/unknown.png';
+
+export const manual: CommandDefinition = {
+    name: 'manual',
+    description: 'Displays image about manual waypoints',
+    category: CommandCategory.A32NX,
```

Comment:
```suggestion
    category: CommandCategory.SUPPORT,
```

You have added this to the support category (at least in folder structure), so we need to define this here too.